src/mudlet.cpp

```diff
@@ -231,6 +231,7 @@ mudlet::mudlet()
 , mpButtonConnect(nullptr)
 , mpActionConnect(nullptr)
 , mpActionDisconnect(nullptr)
+, mpActionCloseProfile(nullptr)
```

Comment:
Actually none of these are necessary - `QPointer<T>` are automatically initialised to the `nullptr` state - when @vadi2 made all those pointers permanent members of the class in https://github.com/Mudlet/Mudlet/commit/90071339341beca72df6aa97fbfc495ebde3eeef (in 2018) they did not have those unneeded initialisations but it seems I got a bit carried away in #4388!

Reply:
I'll leave it as it is for now, I'll remove those in another PR maybe :)

Reply:
Unfortunately, since it IS here it needs to be in the correct place (initialisation list items should be initialised **in the same order as they are declared in the header file**, otherwise one can get problems when a later item makes use of one declared earlier) - as such this provokes a compiler warning so as to prevent problems. Whilst it is not an issue here the various code analyser Bots will also pick up on it.

tl;dr; you need to move this line to 250 so that it is between these two:
```cpp
, mpActionReconnect(nullptr)
, mpActionScripts(nullptr)
```

Reply:
Fixed.